pliers/extractors/text.py

```diff
@@ -313,3 +334,52 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+class SpaCyExtractor(TextExtractor):
+
+    
+    def __init__(self, model='en_core_web_sm', features=None):
+        verify_dependencies(['spacy'])
+        try: 
+            print(model)
+            model_na=model[:2]
+            self.model = spacy.load(model_na)
+            self.features= features
+            print('loaded model: ',self.model )
+        except (ImportError, IOError, OSError) as e:
+            logging.error("Spacy Models ('"+model+"') not found. Downloading and installing")
```

Comment:
This works, but it's conventional and more maintainable to use string formatting for this kind of thing—i.e., something like `"Spacy Model '{}' not found. Downloading and installing".format(model)`